[DOCS] Getting Started with GX (#8143)
great-expectations/great_expectations: docs/docusaurus/docs/tutorials/getting_started/how_to_use_great_expectations_in_databricks.md

---
title: How to Use Great Expectations in Databricks
---

import Prerequisites from '../../deployment_patterns/components/deployment_pattern_prerequisites.jsx'
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import Congratulations from '../../guides/connecting_to_your_data/components/congratulations.md'
import TechnicalTag from '@site/docs/term_tags/_tag.mdx';

> This guide will help you run an end-to-end workflow with Great Expectations in [Databricks](https://databricks.com/).

Reviewer: ```suggestion
Use the information provided here to use Great Expectations (GX) with your [Databricks](https://databricks.com/) instance.
```

> You will:

Reviewer: ```suggestion
To use GX with Databricks, you'll complete the following tasks:
```

  - Load data
  - Instantiate a <TechnicalTag tag="data_context" text="Data Context" />
  - Create a <TechnicalTag tag="datasource" text="Datasource" /> & <TechnicalTag tag="data_asset" text="Data Asset" />
  - Create an <TechnicalTag tag="expectation_suite" text="Expectation Suite" />
  - Validate data using a <TechnicalTag tag="checkpoint" text="Checkpoint" />

## Prerequisites

<Prerequisites>

> - Have completed Databricks setup including having a running Databricks cluster with attached notebook

Reviewer: ```suggestion
- A running Databricks cluster with an attached notebook
```

> - Have access to [DBFS](https://docs.databricks.com/dbfs/index.html)

Reviewer: ```suggestion
- Access to the [Databricks File System (DBFS)](https://docs.databricks.com/dbfs/index.html)
```
